airflow/config_templates/config.yml

```diff
@@ -1945,6 +1945,14 @@
       type: string
       example: ~
       default: ""
+    - name: git_sync_depth
+      description: |
+        Use a shallow clone with a history truncated to the specified number of commits.
+        0 - do not use shallow clone.
+      version_added: ~
```

Comment:
I like the change - @kaxil should we cherry-pick to 1.10.11 ? If so then this version here should be updated.

Reply:
Yeah, we should, looks like a useful change.